### PR TITLE
[Feature] EP support LoRA SFT

### PR DESCRIPTION
**Pull request by kevssim**
# PR type
- [ ] Bug Fix
- [x] New Feature
- [x] Document Updates
- [ ] More Models or Datasets Support

# PR information

EP support LoRA SFT

### CODE REVIEW

File: src/twinkle/model/transformers/strategy/native_fsdp.py [hunk 1]
```diff
@@ -297,6 +358,12 @@ def _detect_ep_expert_names(model: nn.Module) -> Set[str]:
     return candidate_names & actual_param_names
 
 
+def _ep_expert_state_dict_gather_dim(name: str) -> int:
+    if 'lora_B' in name:
+        return 1
+    return 0
```
[the remark this answers is not in the document]
Comment (kevssim) [reply src/twinkle/model/transformers/strategy/native_fsdp.py:369]: PEFT ParamWrapper keeps expert LoRA tensors flattened instead of storing them as [num_experts, ...]: lora_A is [r * num_experts, in] and lora_B is [out, r * num_experts]. EP therefore owns a contiguous expert block on dim 0 for A and dim 1 for B. This is still expert sharding, not LoRA rank parallelism, so the forward pass does not need an EP all-reduce.

**Review by tastelikefeet — APPROVED** [review]
